assets/src/edit-story/components/helpCenter/menu/stories/index.js

```diff
@@ -70,7 +55,7 @@ export const Menu = () => {
       <Bg>
         <Container>
           <TransitionGroup component={null}>
-            {mounted && <HelpCenterMenu key="someKey" />}
+            {mounted && <HelpCenterMenu key="someKey" readTips={[]} />}
```

Comment:
I think readTips is supposed to be an object right? regardless, doesn't really matter for this storybook instance 🤷

Reply:
🤦 yeah you're right